test/image/mocks/typed_array_repr_scatter.json

```diff
@@ -0,0 +1,13 @@
+{
+  "data": [{
+    "type": "scatter",
+    "x":  {"dtype": "float64", "data": [3, 2, 1]},
```

Comment:
Not sure about using `"data"`, `"data"` has a pretty important meaning already for plotly.js. I'd vote for `values`.

Reply:
or maybe just `"v"` as calling a base64 string a set of _values_ sounds wrong.

Reply:
I renamed `data` to `value` (singular) in 5030d3a. Does that work for you? `v` felt too short 🙂